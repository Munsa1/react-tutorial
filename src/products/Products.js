import React from "react";

const Products = ({items = []}) =>{
  if (items.length<0){
    return <div>No Items found</div>
  }else {
    return <div>{items.map((item, index) =>{
            return <h4>{item}</h4>
    })}</div>
  }
}

export default Products
