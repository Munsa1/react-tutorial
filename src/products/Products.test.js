import React from 'react';
import  Renderer  from 'react-test-renderer';
import Products from './Products';

it('rendrers', () =>{
  const tree = Renderer.create(<Products />).toJSON();
  expect(tree).toMatchSnapshot();
})