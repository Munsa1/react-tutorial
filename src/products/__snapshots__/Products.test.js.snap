// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendrers 1`] = `<div />`;
